frontend/src/pages/Skills/SkillsCatalog.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from 'react';
+import { InputAdornment } from '@mui/material';
+import { Search } from '@mui/icons-material';
+
+import { AuthorizedLayout } from 'components/Layout';
+import Loader from 'components/Loader';
+import Skill from 'components/Skill';
+import { NoContent } from 'components/NoContent';
+import { NO_SKILLS } from 'constants/messages';
+
+import { ISkillsPageProps } from './types';
+import {
+  SkillsPageContainer,
+  SearchSkill,
+  SearchWrapper,
+  SkillsWrapper,
+  StyledDivider,
+} from './styled';
+
+const SkillsCatalog: FC<ISkillsPageProps> = ({
```

Comment:
component should have the same name as the folder and  its import
so you can use

 `SkillsCatalog + SkillsCatalogContainer in folder SkillsCatalog`

 or 

`Skills + SkillsContainer in Skills folder`